clang/test/Driver/darwin-embedded-search-paths.c

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: default-cxx-stdlib=libstdc++
```

Comment:
Maybe this should be `!=libc++` instead?

Reply:
I think it's fine. I'm not sure `REQUIRES` has a `!=`

Reply:
I guess if there was a different cxxlib we would want another version of the test to check its paths anyway

Reply:
Sometimes `default-cxx-stdlib` isn't set at all, and so the `REQUIRES` doesn't always work. `REQUIRES` doesn't have a `!=`, but you can do `!(a=b)`. You can also do `!a || a=b`, but it's probably best to cover libc++ and "everything else" instead of just libc++/libstdc++.